webots_ros2_crazyflie/worlds/crazyflie_apartment.wbt

```diff
@@ -0,0 +1,532 @@
+#VRML_SIM R2023a utf8
```

Comment:
It contains a proto referring to version R2023b, so we need to put this version here as well.
```suggestion
#VRML_SIM R2023b utf8
```